ambry-rest/src/main/java/com/github/ambry/rest/PublicAccessLogHandler.java

```diff
@@ -44,11 +48,13 @@ public class PublicAccessLogHandler extends ChannelDuplexHandler {
   private long responseFirstChunkStartTimeInMs;
   private long responseBytesSent;
   private StringBuilder logMessage;
+  private StringMapMessage structuredLogMessage;
   private HttpRequest request;
   private StringBuilder sslLogMessage;
 
   private static final long INIT_TIME = -1;
-  private static final Logger logger = LoggerFactory.getLogger(PublicAccessLogHandler.class);
+  //private static final Logger logger = LoggerFactory.getLogger(PublicAccessLogHandler.class);
```

Comment:
instead of commenting out, please remove lines that you don't need here and at other places.

Reply:
That is done.